business/istio_validations.go

```diff
@@ -269,6 +271,10 @@ func (in *IstioValidationsService) GetIstioObjectValidations(ctx context.Context
 		objectCheckers = []ObjectChecker{
 			checkers.K8sReferenceGrantChecker{K8sReferenceGrants: istioConfigList.K8sReferenceGrants, Namespaces: namespaces},
 		}
+	case kubernetes.K8sTCPRoutes:
```

Comment:
this case should be included, even empty, this is required by validations logis, otherwise error is shown as unknown object type. But yes this is left for the future, as the support is experimental.